src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -206,12 +206,31 @@ void IslandWindow::OnRestore()
     // TODO MSFT#21315817 Stop rendering island content when the app is minimized.

```

Comment:
I know it's not part of this PR, but did this MSFT get ported to a GH number and can you substitute it?